src/app.ts

```diff
@@ -13,14 +13,35 @@ interface TemperatureSummary {
   average: number
 }
 
+const data: TemperatureReading[] = []
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  readings.forEach((item) => {
```

Comment:
You can use a more descriptive name instead of `item`, I think `reading` is a better name